limitador-server/src/envoy_rls/server.rs

```diff
@@ -562,9 +556,7 @@ mod tests {
     }
 
     #[tokio::test]
-    async fn test_0_hits_addend_is_converted_to_1() {
-        // "hits_addend" is optional according to the spec, and should default
-        // to 1, However, with the autogenerated structs it defaults to 0.
```

Comment:
Well... that's a problem! Now it'd be fine _for us always_, as we do explicitly set it tho.

Reply:
Well the proto seems to say it's not optional (or is it because it's the last field?!), so it has to be set, hasn't it?!